engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/semantic/PanicsTest.scala

```diff
@@ -49,6 +49,29 @@ class PanicsTest extends InterpreterTest {
       consumeOut shouldEqual List("(Error: MyError)")
     }
 
+    "message should be computed soon enough" in {
+      val code =
+        """from Standard.Base import all
+          |
+          |type LM
+          |    V txt
+          |
+          |    to_display_text self =
+          |        res = self.txt + " from exception"
+          |        Panic.throw res
+          |
+          |main = Panic.throw (LM.V "Hi")
+          |""".stripMargin
+
+      try {
+        eval(code)
```

Comment:
`eval(code)` is expected to throw `InterpreterException`, right? Shouldn't you add `fail("Exception expected")` after this line then?

Reply:
Done in 7c0472b